src/main/java/org/sagebionetworks/bridge/hibernate/HibernateAdherenceReportDao.java

```diff
@@ -0,0 +1,79 @@
+package org.sagebionetworks.bridge.hibernate;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static org.sagebionetworks.bridge.models.SearchTermPredicate.AND;
+
+import java.util.List;
+
+import javax.annotation.Resource;
+
+import org.sagebionetworks.bridge.dao.AdherenceReportDao;
+import org.sagebionetworks.bridge.hibernate.QueryBuilder.WhereClauseBuilder;
+import org.sagebionetworks.bridge.models.AccountTestFilter;
+import org.sagebionetworks.bridge.models.PagedResourceList;
+import org.sagebionetworks.bridge.models.schedules2.adherence.weekly.WeeklyAdherenceReport;
+import org.springframework.stereotype.Component;
+
+@Component
+public class HibernateAdherenceReportDao implements AdherenceReportDao {
+
+    static final String COMPLIANCE_UNDER_FIELD = "complianceUnder";
+    static final String LABEL_FILTER_FIELD = "labelFilter";
+    static final String STUDY_ID_FIELD = "studyId";
+    static final String APP_ID_FIELD = "appId";
+    
+    private HibernateHelper hibernateHelper;
+
+    @Resource(name = "mysqlHibernateHelper")
+    final void setHibernateHelper(HibernateHelper hibernateHelper) {
+        this.hibernateHelper = hibernateHelper;
+    }
+
+    @Override
+    public void saveWeeklyAdherenceReport(WeeklyAdherenceReport report) {
+        checkNotNull(report);
+        
+        report.setClientTimeZone("v1_apps_appId_studies_studyId_participants_searchv1_apps_appId_studies_studyId_participants_searchv1_apps_appId_studies_studyId_participants_searchv1_apps_appId_studies_studyId_participants_searchv1_apps_appId_studies_studyId_participants_searchv1_apps_appId_studies_studyId_participants_search");
```

Comment:
This is an odd time zone. Is the intent to overwrite with the participant's time zone?